pkgs/by-name/ch/chance/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  blueprint-compiler,
+  cargo,
+  desktop-file-utils,
+  fetchFromGitLab,
+  libadwaita,
+  meson,
+  ninja,
+  nix-update-script,
+  pkg-config,
+  rustPlatform,
+  rustc,
+  stdenv,
+  wrapGAppsHook4,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "chance";
+  version = "4.0.0";
+
+  src = fetchFromGitLab {
+    owner = "zelikos";
+    repo = "rollit";
+    rev = finalAttrs.version;
+    hash = "sha256-2lZ7iMHMFE1wTSlJj0mIUV62jO0NundYiOC8rdUJGkQ=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit (finalAttrs) src;
+    name = "chance-${finalAttrs.version}";
```

Comment:
```suggestion
    inherit (finalAttrs) pname version src;
```

Reply:
It's a pretty common pattern, @Aleksanaa treewide pull request soon? :P 
https://github.com/search?q=repo%3ANixOS%2Fnixpkgs+cargoDeps+%3D+rustPlatform.fetchCargoTarball+name+%3D+%22%24%7Bpname%7D-%24%7Bversion%7D%22&type=code